src/AnimatedSprite.cpp

```diff
@@ -81,24 +87,50 @@ sf::FloatRect AnimatedSprite::getGlobalBounds() const
 
 void AnimatedSprite::setHighlight(bool h)
 {
-    if (h)
-    {
-        _highlight.setPosition(getPosition());
-        auto b = getGlobalBounds();
-        _highlight.setSize(sf::Vector2f{ b.width, b.height });
-    }
-    else
+    _isHighlighted = h;
+    setHighlightPosition();
+}
+
+void AnimatedSprite::draw(  sf::RenderTarget& target, 
+                            sf::RenderStates states     ) const
+{
+
+    states.transform *= getTransform();
+    target.draw(_sprite, states);
+
+    if(_isHighlighted)
     {
-        _highlight.setSize(sf::Vector2f{ 0.f, 0.f });
+        target.draw(_highlight);
     }
 
 }
 
-void AnimatedSprite::draw(sf::RenderTarget & target, sf::RenderStates states) const
+void AnimatedSprite::setHighlightPosition()
 {
-    states.transform *= getTransform();
-    target.draw(_sprite, states);
-    target.draw(_highlight);
+    auto b = getGlobalBounds();
+    
+    float x = b.left + ((b.width / 2)   - (DEFAULT_HIGHLIGHT_WIDTH / 2));
+    float y = b.top + ((b.height)       - (DEFAULT_HIGHLIGHT_HEIGHT));
+    
+    _highlight.setPosition( sf::Vector2f{ x, y } );
+    
+    _highlight.setSize(sf::Vector2f{DEFAULT_HIGHLIGHT_WIDTH,
```

Comment:
I'd like for us to figure out a way to highlight the entire object when we want, or just a portion when we want. It seems like `Player` might be the only object where we want only a portion of the object highlighted.